chef_master/source/resource_yum.rst

```diff
@@ -472,9 +472,9 @@ or:
 
 .. tag resource_package_handle_cookbook_file_and_package
 
-.. To handle cookbook_file and yum_package when both called in the same recipe
```

Comment:
This one we don't actually want to replicate. This actually should be yum_package since it's in the yum resource page